tests/test_pqserver.py

```diff
@@ -52,12 +56,95 @@ def db(tmpdir, scope="module"):
     return db_path
 
 
+@pytest.fixture
+def astropy_tables(scope="module"):
+    t = dict()
+    return t
+
+
+@pytest.fixture
+def lsstcomcamsim(tmpdir, astropy_tables, scope="module"):
+    schema = "cdb_lsstcomcamsim"
+    db_path = tmpdir / "test.db"
+    schema_path = tmpdir / f"{schema}.db"
+
+    data_path = Path(__file__).parent / "lsstcomcamsim"
+    sql_path = data_path / f"{schema}.sql"
+
+    # Build the main db file to specify where to look for
+    # specific schemas.
+    with sqlite3.connect(db_path) as conn:
+        conn.execute("CREATE TABLE schemas (name text, path text)")
+        conn.execute(f"INSERT INTO schemas VALUES ('{schema}', '{schema_path}')")
+
+    # Set up the lsstcomcamsim schema with the schema from the SQL file
+    # generated with felis, and with sample data interrogated from consdb,
+    # and with fake made up flex data
+    with sqlite3.connect(schema_path) as conn:
+        conn.executescript(sql_path.read_text())
+    import time
```

Comment:
Will you move this import to the top of the module?

Reply:
Probably shouldn't be there at all.

Reply:
you use time.sleep() two lines below this. Don't forget to remove that too, if that is your intention.